app/views/iterations/edit.html.haml

```diff
@@ -54,5 +54,7 @@
     %br
 
     %br
-    = submit_tag 'Save Changes'
+    = submit_tag 'Save Changes', :class => 'btn btn-success pull'
```

Comment:
can you explain what 'pull' class does??

Reply:
most of the submit buttons in other view files had "pull" which I thought was necessary to request changes in the database. I ran the command without pull and it worked the same, so I deleted pull from the command. Thanks a lot for the suggestion!